projects/driver_controls/src/fsm/direction_fsm.c

```diff
@@ -40,51 +49,52 @@ static bool prv_check_neutral(const Event *e) {
   }
 }
 
-static bool prv_check_forward(const Event *e) {
+static bool prv_guard_forward(const Event *e) {
   // Powering off while the car is moving can be dangerous. As a result, powering off is
```

Comment:
Technically the car has to turn off by regulation if there is a fault, right? Understandably this is from the perspective of driver controls but Power Distribution can override this if fault conditions arise.

Reply:
It should cause a BPS fault message, so I can transition the power FSM on that.